ajax/libs/vue-instant/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "vue-instant",
+  "filename": "vue-instant.browser.js",
+  "description": "vue instant allows you to easily create custom search controls with auto suggestions for your vue 2 application.",
+  "version": "1.0.2",
+  "author": "santiago <santiago.blanco.vilchez@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/santiblanko/vue-instant.git"
+  },
+  "keywords": [
+    "vue",
+    "vuejs",
+    "plugin",
+    "autocomplete",
+    "autosuggest",
+    "instant search",
+    "instant",
+    "searchbox",
+    "component"
+  ],
+  "homepage": "https://santiblanko.github.io/vue-instant/",
+  "npmName": "vue-instant",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(vue-instant.common.js)"
```

Comment:
Let's use `**/!(vue-instant.common.js)` instead, it can fetches files under sub-directories compared to the original one.